### PR TITLE
chore(cohere): update doc

### PR DESCRIPTION
**Pull request by chuang8511**
Because

- we add cohere into vdp

This commit

- add the pricing for cohere

### CODE REVIEW

File: docs/cloud/credit.zh-CN.mdx [hunk 1]
```diff
@@ -103,6 +108,13 @@ Credit** cost per unit.
 | OpenAI | TTS 1    | 150                           |
 | OpenAI | TTS 1 HD | 300                           |
 
+
+#### Text Reranking
+| Vendor    | Model             | Credit Cost Per 1000 Searches      |
+| :-------- | :---------------- | :--------------------------------- |
+| Cohere    | Rerank 3          | 20                                 |
```
Comment (donch1989) [inline docs/cloud/credit.zh-CN.mdx:115]: @chuang8511 
This number looks not correct.
Should it be 20,000?

Comment (chuang8511) [reply docs/cloud/credit.zh-CN.mdx]: @donch1989 
Sync
I am adding MistralAI pricing in this PR.

Comment (chuang8511) [reply docs/cloud/credit.zh-CN.mdx]: Done.

**Comment by chuang8511** [comment]
@GeorgeWilliamStrong
Just a reminder.
MistralAI & Cohere will be able to use with Instill Credit in this week.
So, we will need to deploy this doc in this week as well.

Thank you!

**Review by GeorgeWilliamStrong — APPROVED** [review]
LGTM

**Pull request merged**